test/unit/app/services/bulk_republisher_test.rb

```diff
@@ -232,6 +232,15 @@ class BulkRepublisherTest < ActiveSupport::TestCase
       end
     end
 
+    context "for editionable content types, like Contact attached to a WorldwideOrganisation" do
```

Comment:
This description confuses me a little, especially since we already have context about editionable types, which are handled differently. The changes here seem to be about non-editionable types. Should the contexts be something like this?
- for editionable content types, like CaseStudy (no change)
- **for non-editionable content types, like Contact, when publishable to Publishing API**
- **for non-editionable content types, like Contact, when not publishable to Publishing API**
- for non-republishable model types (no change: though we should probably change this to content types)
- for non-existant content types (no change)

Reply:
That's much better than what I wrote, I've updated these.